src/server/external-services/hgnc.js

```diff
@@ -0,0 +1,61 @@
+const fetch = require('node-fetch');
+const _ = require('lodash');
+const { HGNC_BASE_URL } = require('../../config');
+const { EntitySummary, DATASOURCES } = require('../../models/entity/summary');
+
+//Could cache somewhere here.
+const fetchBySymbol = symbol => {
+  return fetch( `${HGNC_BASE_URL}/fetch/symbol/${symbol}`,
+    { headers: {
+      'Accept': 'application/json'
+      }
+    })
+    .then(res => res.json());
+};
+
+const fetchBySymbols = async symbols => {
+  const docs = symbols.map( async symbol => {
+    // Process each symbol individually
+    const result = await fetchBySymbol( symbol );
+    return result;
+  });
+  return await Promise.all( docs );
+};
+
+const getEntitySummary = async symbols => {
+
+  const summary = {};
+  if ( _.isEmpty( symbols ) ) return summary;
+
+  const results = await fetchBySymbols( symbols );
+  const nonEmptyResults = _.filter( results, o => _.get( o, 'response.numFound') );
```

Comment:
It's fine to use async/await on the server like this example, but do not do that on the client.  The polyfill for that is large, and [it's still a relatively new feature](https://caniuse.com/#search=await).

Reply:
> You should use array map and array filter instead of lodash map/filter.

What's the difference between the two?

Reply:
`_.filter()` can be more useful than `array.filter()` if you're doing functional programming.  I don't really have a preference anymore, since we're using lodash so extensively overall in this project.